### PR TITLE
SSB: Add Cake, Hubriz, and more

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Tested and works

File: mods/seasonal/abilities.js [hunk 1]
```diff
@@ -331,10 +341,17 @@ let BattleAbilities = {
 	},
 	// torkool
 	deflectiveshell: {
-		desc: "Non-contact moves do 33% less damage to this pokemon.",
-		shortDesc: "Non-contact moves do 33% less damage to this pokemon.",
+		desc: "Non-contact moves do 33% less damage to this pokemon. Summons Sunny Day on switch-in.",
+		shortDesc: "Non-contact moves do 33% less damage; summons Sun.",
```
Comment (HoeenCoder) [inline mods/seasonal/abilities.js]: Too long, "Sun." is cut off. Use this:

`Drought + Non-contact does 33% less damage.` (correct spelling if needed)

File: mods/seasonal/random-teams.js [hunk 2]
```diff
@@ -101,6 +107,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: 'Food Rush',
 				evs: {hp: 4, atk: 252, spd: 252}, nature: 'Adamant',
 			},
+			'Bimp': {
+				species: 'Stakataka', ability: 'Learn Something New!', item: 'Rockium Z', gender: 'M',
+				moves: ['Gyro Ball', 'Stone Edge', 'Earthquake'],
+				signatureMove: 'Trivia Room',
+				evs: {hp: 252, atk: 252, spd: 4}, ivs: {spe: 0}, nature: 'Brave',
```
Comment (HoeenCoder) [inline mods/seasonal/random-teams.js]: EVs: 248 HP/252 ATK/8 SPD

File: mods/seasonal/moves.js [hunk 3]
```diff
@@ -1199,6 +1276,38 @@ let BattleMovedex = {
...
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Petal Blizzard", target);
+			this.add('-anim', source, "Leaf Tornado", target);
+		},
+		secondary: {
+			chance: 20,
+			onHit: function (target, source) {
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:1299]: This works right? I'm p sure it does but just checking that its tested.

Comment (ghost) [reply mods/seasonal/moves.js:1299]: Yes

**Pull request merged**